.spelling

```diff
@@ -701,6 +701,7 @@ wildcard
 wildcarded
 wildcards
 workgroup
+WorkloadEntry
```

Comment:
please use backticks in the documentation for `WorkloadEntry` and remove this entry from the `.spelling` file as per various documentation guides.

Reply:
done